<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NWFOS </string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Dialog{
	background-color: #fff;
}</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>1281</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>250</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>250</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Gambar</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Harga</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Deskripsi</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelHarga">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#labelHarga{
	border-radius: 10px;
	background-color: rgb(225, 225, 225);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jumlah : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="jumlahText">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#jumlahText{
	border-radius: 10px;
	background-color: rgb(240, 240, 240);
}

QLineEdit#jumlahText::focus{
	border: 2px solid blue;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addToCartButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#addToCartButton {
    background-color: #56BB86;
    border-radius: 10px;
    font:14px;
	color:#fff;
    padding: 6px;
}
QPushButton#addToCartButton:hover {
    background-color:rgb(46, 209, 119);
	border-style: inset;
}

QPushButton#addToCartButton:pressed {
    background-color:rgb(58, 127, 90);
	border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Add To Cart</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNama">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#labelNama{
	border-radius: 10px;
	background-color: rgb(225, 225, 225);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>1281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame {
	background-color: rgb(86, 187, 134);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="gtCartButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>20</y>
      <width>112</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#gtCartButton {
    background-color: rgb(225, 225, 225);
    border-radius: 10px;
    font:14px;
	color:#000;
    padding: 6px;
}
QPushButton#gtCartButton:hover {
	background-color:rgb(231, 231, 231);
    border-style: inset;
}
QPushButton#gtCartButton:pressed {
	background-color:rgb(190, 190, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Go to cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allMenuButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>112</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#allMenuButton {
    background-color: rgb(225, 225, 225);
    border-radius: 10px;
    font:14px;
	color:#000;
    padding: 6px;
}
QPushButton#allMenuButton:hover {
	background-color:rgb(231, 231, 231);
    border-style: inset;
}
QPushButton#allMenuButton:pressed {
	background-color:rgb(190, 190, 190);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>All Menu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchText">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#searchText{
	border: 1px solid black;
	border-radius: 15px;
	padding:3px;
}

QLineEdit#searchText::focus{
	border: 2px solid blue;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Search Here</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nama Makanan</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Harga (Rp)</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>tableWidget</zorder>
  <zorder>labelHarga</zorder>
  <zorder>label_3</zorder>
  <zorder>jumlahText</zorder>
  <zorder>addToCartButton</zorder>
  <zorder>labelNama</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
